Features/movie page api data

WojciechBylica opened the pull request:
the first step of APi data providing.
additional correction of exports in some components

KarolCiesluk reviewed (approved):
Great job Wojtek! 👍

patrycja-i-lesniak reviewed (approved):
The code looks fine at the moment.

WojciechBylica commented on src/features/MovieDetails/index.js:
```diff
@@ -3,25 +3,35 @@ import { Wrapper } from "./styled";
+                        </Wrapper>
+                    </>);
+            case "error":
+                return (<Error />);
+            default:
+                return <NoResults />;
+        };
```
corrected

WojciechBylica commented on src/features/MovieDetails/useAPIData.js:
```diff
@@ -12,19 +12,24 @@ export const useAPIData = () => {
                 const movieDetailsAPI = `${APIAdress}movie/${movieId}?${APIKey}`;
                 const response = await axios.get(movieDetailsAPI);
-                const response2 = await axios.get(APIConfigurarion);
-                console.log(response2);
+
+                const creditsAPI = `${APIAdress}movie/${movieId}/credits?${APIKey}`;
+                const response2 = await axios.get(creditsAPI);
+
                 setMovieData({
                     status: "success",
-                    movieData: response
+                    movieData: response,
+                    creditsData: response2
                 });
```
done

WojciechBylica commented on src/features/MovieDetails/useAPIData.js:
```diff
@@ -12,19 +12,24 @@ export const useAPIData = () => {
-                const movieId = '20';
-                const APIConfigurarion = `https://api.themoviedb.org/3/configuration?${APIKey}`;
-                // const  movieKeyWordsAPI = `https://api.themoviedb.org/3/movie/18/keywords?api_key=${APIKey}`;
+                const movieId = '21';
```
lets use ""

WojciechBylica commented:
added all needed API requests
movied useAPIData.js to src
updated PersonList to show the results of API data working
temporary moved MovieDetails to route path=Movies to check working of navigation
